scripts/functions.sh

```diff
@@ -55,6 +55,8 @@ done
 export BASE_DIR=${BASE_DIR:-"/opt/rpc-openstack"}
 export HOST_RCBOPS_DOMAIN="rpc-repo.rackspace.com"
 export HOST_RCBOPS_REPO=${HOST_RCBOPS_REPO:-"http://${HOST_RCBOPS_DOMAIN}"}
+export RPCO_APT_ARTIFACT_ENABLED=${RPCO_APT_ARTIFACT_ENABLED:-"yes"}
```

Comment:
For the sake of consistency these variables should be RPC_ not RPCO_

Reply:
Done.